x11-misc/sddm/sddm-0.19.0_p20210418.ebuild

```diff
@@ -0,0 +1,173 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Maybe you want to change to EAPI=8 and to a newer snapshot.
This is almost 4 months old.

Reply:
EAPI-8 is not possible at this point, new snapshot will be much more experimental with recent changes.